src/main/resources/com/rultor/agents/req/merge.sh

```diff
@@ -20,6 +20,12 @@ while [ $(git show-branch "${BRANCH}" 2>/dev/null | wc -l) -gt 0 ]; do
 done
 
 git checkout -B "${BRANCH}" "fork/${fork_branch}"
+git fetch origin
+if git log --name-status --oneline HEAD..origin/${head_branch} | grep ".rultor.yml"; then
+  echo "Since original repository has different version of .rultor.yml file you have to sync your fork with the upstream repository."
+  exit 1
+fi
```

Comment:
@original-brownbear 

> Getting the merge result is actually fairly easy and can be accomplished in two different ways I can think of right now

Problem is not to merge forks. It is easy task. Problem is to be sure, that we have correct version of `.rultor.yml` here: https://github.com/yegor256/rultor/pull/1064/files#diff-eb31f19e43e00bbef161a0fbd2a228a0R93 . And in current architecture of Rultor you cannot merge `.rultor.yml` before creating profile. So, the only way here - sync fork with upstream.

> Rultor does not solve any conflicts btw, this needs to be done by the author. All we really need to ensure is that we are actually building against the resulting .rultor.yml.

Yes, this is what I speaking about. Let fork author to do it himself.